src/modules/shared/components/Sidebar/Sidebar.tsx

```diff
@@ -0,0 +1,82 @@
+import { Sidebar, Menu, MenuItem } from "react-pro-sidebar";
+import { Link } from "react-router-dom";
+import { useState } from "react";
+export default function SideBar() {
+  const [isCollapse, setIsCollapse] = useState(true);
+  const toggleCollapse = () => {
+    setIsCollapse(!isCollapse);
+  };
+  return (
+    <>
+      <div className="sidebar-container">
+        <Sidebar collapsed={isCollapse} className="rounded-R">
+          <span
```

Comment:
This should be a button, and we need to add a text that describes the action for screen readers  as we did in the session with the sr-only class

Reply:
done hope you review the new code